content/en/docs/concepts/policy/_index.md

```diff
@@ -1,11 +1,68 @@
 ---
 title: "Policies"
 weight: 90
+no_list: true
 description: >
-  Policies you can configure that apply to groups of resources.
+  Manage security and best-practices with policies.
 ---
 
-{{< note >}}
-See [Network Policies](/docs/concepts/services-networking/network-policies/)
-for documentation about NetworkPolicy in Kubernetes.
-{{< /note >}}
+<!-- overview -->
+
+Kubernetes policies are configurations that manage other configurations or runtime behaviors. Kubernetes offers various forms of policies, described below:
```

Comment:
```suggestion
Kubernetes policies are means for you to constrain or require specific outcomes,
such as enforcing how a (Kubernetes) API is used, or restricting behavior at runtime.

Kubernetes offers various forms of policies, described below:
```
?

Reply:
- `specific outcomes` seems vague here....can we state as `specific configurations or outcomes`?
- `such as enforcing how a (Kubernetes) API is used` --> `such as enforcing how a resource is used`?